lib/swagger-mock-validator/validate-spec-and-mock/validate-parsed-mock-request-body.ts

```diff
@@ -23,6 +23,12 @@ const transformSchema = (
         }
     });
 
+    // draft-06 onwards converts exclusiveMinimum and exclusiveMaximum to numbers
+    traverseJsonSchema(modifiedSchema, (mutableSchema) => {
+        mutableSchema.exclusiveMaximum = mutableSchema.exclusiveMaximum ? mutableSchema.maximum : undefined;
```

Comment:
Is it possible maximum/minimum here could not be defined or would it fail the AVJ parsing before this part of the code runs? (ditto in related places)